NDTensors/test/combiner.jl

```diff
@@ -22,12 +23,12 @@ using ITensors: QN, Index
       @test output_tensor isa DenseTensor
       @test dims(output_tensor) == output_tensor_inds
       for i in 1:length(input_tensor)
-        @test input_tensor[i] == output_tensor[i]
+        @test @allowscalar input_tensor[i] == output_tensor[i]
```

Comment:
Maybe better to annotate the for loop with `@allowscalar` instead, I notice you use that style in other places.

In general, it is good to try to choose a style or convention to follow for things like that and stick to it, to keep the code consistent and make it clearer what to do when adding new code.

Reply:
I went through and looked at all the allowscalar to try and be consistent with this, thanks!